### DISCUSSION TITLE
In Designer component, the Add new field and Left Sidebar for Placing Schemas is not visible.

### DISCUSSION THREAD

**Discussion by rubixgithub** [pdfme/pdfme#471] [Q&A]
I'm using this versions
"@pdfme/common": "^4.0.0-alpha.0"
"@pdfme/generator": "^4.0.0-alpha.0"
"@pdfme/schemas": "^4.0.0-alpha.0"
"@pdfme/ui": "^4.0.0-alpha.0"

![Screenshot (612)](https://github.com/pdfme/pdfme/assets/118471532/bae066d4-8b2c-4292-b0ed-d25cb5ca490a)

**Comment by hand-dot** [comment]
@rubixgithub Could you give me the reproducible repo?

**Reply by rubixgithub** [reply]
I cannot because this is private project. But I tried the same code from another codebase and it worked. I don't know what is the problem.

**Reply by rubixgithub** [reply]
For better understanding I can provide my all dependencies where this issue arise.

"dependencies": {
    "@emotion/react": "^11.11.3",
    "@emotion/styled": "^11.11.0",
    "@fortawesome/fontawesome-svg-core": "^6.4.2",
    "@fortawesome/free-solid-svg-icons": "^6.4.2",
    "@fortawesome/react-fontawesome": "^0.2.0",
    "@mui/icons-material": "^5.11.16",
    "@mui/material": "^5.15.3",
    "@mui/styled-engine-sc": "^5.12.0",
    "@mui/x-data-grid": "^6.18.7",
    "@mui/x-date-pickers": "^6.4.0",
    "@nivo/bar": "^0.85.1",
    "@nivo/geo": "^0.85.1",
    "@nivo/line": "^0.85.1",
    "@pdfme/common": "^4.0.0-alpha.0",
    "@pdfme/generator": "^4.0.0-alpha.0",
    "@pdfme/schemas": "^4.0.0-alpha.0",
    "@pdfme/ui": "^4.0.0-alpha.0",
    "apexcharts": "^3.41.1",
    "axios": "^1.4.0",
    "crypto-js": "^4.2.0",
    "d3": "^7.8.5",
    "date-fns": "^2.30.0",
    "dayjs": "^1.11.10",
    "gsap": "^3.12.5",
    "html-react-parser": "^3.0.16",
    "laravel-vite-plugin": "^0.8.0",
    "lodash": "^4.17.21",
    "material-ui-popup-state": "^5.0.9",
    "prop-types": "^15.8.1",
    "react": "^18.2.0",
    "react-cookie": "^6.1.1",
    "react-dom": "^18.2.0",
    "react-feather": "^2.0.10",
    "react-hook-form": "^7.45.4",
    "react-hot-toast": "^2.4.1",
    "react-icons": "^4.12.0",
    "react-jwt": "^1.2.0",
    "react-router-dom": "^6.11.1",
    "redux-devtools-extension": "^2.13.9",
    "styled-components": "^5.3.10",
    "uuid": "^9.0.1",
    "zustand": "^4.4.4"
  },
  "devDependencies": {
    "@types/react": "^18.2.15",
    "@types/react-dom": "^18.2.7",
    "@vitejs/plugin-react-swc": "^3.3.2",
    "autoprefixer": "^10.4.16",
    "eslint": "^8.45.0",
    "eslint-plugin-react": "^7.32.2",
    "eslint-plugin-react-hooks": "^4.6.0",
    "eslint-plugin-react-refresh": "^0.4.3",
    "postcss": "^8.4.31",
    "tailwindcss": "^3.3.3",
    "vite": "^5.2.0"
  }
